snooty/postprocess.py

```diff
@@ -2427,6 +2471,22 @@ def find_toctree_nodes(
                 visited_file_ids,
             )
 
+    @staticmethod
+    def multi_page_tutorials(
+        tree: Dict[str, SerializableType], multi_page_tutorials: List[str]
+    ) -> Dict[str, n.SerializedNode]:
```

Comment:
Not-blocking: I know that python will understand the definitions, but it makes me nervous to have the method name and a parameter name be the same.

Reply:
Good point! It would help a bit with readability too. Updated function to be `generate_multi_page_tutorials`.